content/doc/developer/security/misc.adoc

```diff
@@ -134,3 +134,8 @@ Many Jenkins plugins allow the use of Groovy to extend their capabilities.
 For example, the plugin:email-ext[Email Extension Plugin] allows executing a script to determine whether an email should be sent.
 
 All such functionality, if available to users without Overall/Administer, must integrate with plugin:script-security[Script Security Plugin].
+
+=== Logging secrets
+
+When there is a need to log secrets, the logging level should be chosen wisely.
```

Comment:
Seems worded too generically. It doesn't introduce a more complex concept, so might as well be
```suggestion
When there is a need to log secrets, they must be logged on a level not visible by default in the system log.
```

(with corresponding changes in the next sentence, which is now partially redundant.)

Reply:
I tried to rephrase the sentence, what do you think about this new one?